development.ini

```diff
@@ -15,8 +15,15 @@ pyramid.includes =
     pyramid_debugtoolbar
     pyramid_celery
 
-git.path = %(here)s/repo/
-git.content_repo_url = https://github.com/universalcore/unicore-cms-content-ffl-tanzania
+# git.path can be a local path or a URL that points to a unicore.distribute repo endpoint.
+# The index on Elasticsearch corresponds to the basename (minus .git or .json suffix).
+# For that reason, the basename must be alphanumeric and lowercase. It may also not
+# start with _.
+git.path = %(here)s/repo
```

Comment:
I tried that, but `git.path` and `git.content_repo_url` are used by eg-tools commands. So I thought it best to leave them as they are so that we still have all those commands at our disposal.